src/_playground/App.scss

```diff
@@ -571,3 +571,134 @@ a.toc-link {
         }
     }
 }
+
+
+
+$arrow-height: 8px;
+$arrow-width: 13px;
+$arrow-width-half: $arrow-width/2;
+
+.fd-popper__body {
+    border: solid 1px #89919a;
+    position: absolute;
+    white-space: nowrap;
+    z-index: 2;
+    border-radius: 4px;
+    background-color: var(--fd-color-background-2);
+    box-shadow: 0 5px 20px 0 #d9d9d9, 0 2px 8px 0 #eeeeef;
+    box-shadow: 0 5px 20px 0 var(--fd-color-neutral-3), 0 2px 8px 0 var(--fd-color-neutral-2);
+    opacity: 1;
+
+    &--no-arrow {
```

Comment:
In the past I have disliked this method of dynamically generating classnames, because it can be missed in future refactors. I see fiori fundamentals uses it quite a bit. 

Should we allow or disallow this practice?

Reply:
Yeah, I could go either way.  I know it makes searching for exact matches harder, but if we don't do any of these, then it almost seems like we've lost one of the big advantages of sass.

Reply:
I added a "Use of sass to build full class names" item to the [Topics for Fiori Fundamentals](https://github.com/SAP/fundamental-react/wiki/Topics-for-Fiori-Fundamentals}.